.ci/lib.sh

```diff
@@ -40,6 +40,10 @@ install_yq() {
 	"x86_64")
 		goarch=amd64
 		;;
+
+	"ppc64le")
+		goarch=ppc64le
+                ;;
```

Comment:
you're still missing a blank line below this one to match the other case statements ;-)

Reply:
Okay, I realised that before sending the original patch, but for some reason, I was just maintaining what was already there. 
https://github.com/kata-containers/packaging/blob/7705d68f7dadc189c3b836ad5972056ab8e881fb/.ci/lib.sh#L43

Reply:
I shall resend the patch.

Reply:
I hate to say this, but this is *still* inconsistent with what is already there. The two semi-colons aren't aligned like they are in the other two cases.

Reply:
@jodh-intel : Now? :-(